src/main/resources/application.yml

```diff
@@ -27,7 +28,7 @@ jobs:
     batch-size: 100
   process-print-responses:
     name: "ProcessPrintResponses"
-    cron: "0 0 */1 * * *" # every hour by default
+    cron: "0 */15 * ? * *" # every 15 minutes by default
```

Comment:
@NathanRussellValtech hope this cron is ok? Can update to hourly if needed later on